deploy/helm/azure-industrial-iot/values.yaml

```diff
@@ -181,6 +181,18 @@ azure:
       # azure.auth.clientsApp.secret is client secret (password) of AAD App Registration for ClientsApp.
       secret: null
 
+  # Those are details used to create Tsi url.
+  tsi:
+     # Data Access FQDN
+     url: null
+
+  # Those are details used to create workbook url.
+  logAnalyticsWorkbook:
+    # azure.logAnalyticsWorkbook.workbookId is the workbook id of Log Analytics Workbook instance.
+    workbookId: null
```

Comment:
@dacolgit, after checking how the whole workbook URL is generated I now have a question regarding how can a client actually get the value of `workbookId` ? Because for most of the configuration values here I provide a documentation in `deploy\helm\azure-industrial-iot\README.md` on how to get the value for an existing or newly created Azure resource. Can you please check it and see if there is a similar `az` command to get a Workbook id ? If there isn't one, then I would actually suggest to remove `workbookId` from the Helm chart as it will only introduce a confusion about what it is and how to get it without actually providing much in terms of functionality.

Here I want to also note, that if we have the link as a PCS variable (or actually pieces required to assemble it) and we save it in the KeyVault then the Engineering Tool will automatically get this value as it has a KeyVault configuration provider. This means that if we have all the information in KeyVault, then without adding anything in the Helm chart Engineering Tool will already be able to show the workbook link.

Reply:
I removed the part related to helm chart